charts/pvc-autoresizer/templates/controller/clusterrole.yaml

```diff
@@ -1,34 +1,42 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: {{ template "pvc-autoresizer.fullname" . }}-controller
+  name: {{ template "pvc-autoresizer.fullname" . }}-controller-storageclasses
   labels:
     {{- include "pvc-autoresizer.labels" . | nindent 4 }}
 rules:
 - apiGroups:
-  - ""
+  - storage.k8s.io
   resources:
-  - events
+  - storageclasses
   verbs:
-  - create
   - get
   - list
   - watch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ template "pvc-autoresizer.fullname" . }}-controller-persistentvolumeclaims
+  labels:
+    {{- include "pvc-autoresizer.labels" . | nindent 4 }}
+rules:
 - apiGroups:
   - ""
   resources:
   - persistentvolumeclaims
   verbs:
   - get
   - list
+  - watch
   - patch
   - update
-  - watch
 - apiGroups:
-  - storage.k8s.io
+  - ""
   resources:
-  - storageclasses
+  - events
```

Comment:
I would like you not to change the order of resources. It makes the code changes harder to understand.

Original order of resources (other than `storageclasses`) was as follows.
```
rules:
- apiGroups:
  - ""
  resources:
  - events
 ...
- apiGroups:
  - ""
  resources:
  - persistentvolumeclaims
 ```

So please locate the `persistentvolumeclaims` after `events` as it is.

Reply:
Reordered so that `events` is first entry in list followed by `persistentvolumeclaims` list entry (see fixup commit).